### ISSUE TITLE
Failed to install version 1.0.2

### ISSUE THREAD

**Issue by gtchance** [blargity/react-native-schemes-manager#40]
### Steps to reproduce the behavior
Set version to 1.0.2, run npm install

### Expected behavior
react-native-schemes-manager installs to node_modules

### Actual behavior
Fails with error:
```
.../node_modules/react-native-schemes-manager/src/fix-script.js:20
			const scriptSettings = mappings.settings['fix-script'] || {};
			                                        ^

TypeError: Cannot read property 'fix-script' of undefined
```

**Comment by adebree** [comment]
Adding an empty 'settings' object to xcodeSchemes in package.json fixes the failure as well as downgrading to 1.0.1

**Comment by henrikra** [comment]
@tylerhou Can you fix this?

**Comment by blargity** [comment]
This should now be resolved in v1.0.3. I'll close this but definitely feel free to lodge a new issue if it's still happening.
